packages/matter.js/src/cluster/definitions/AccessControlCluster.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*** THIS FILE IS GENERATED, DO NOT EDIT ***/
+
+import { WritableAttribute, AccessLevel, OptionalWritableAttribute, FixedAttribute, Event, EventPriority } from "../../cluster/Cluster.js";
+import { TlvArray } from "../../tlv/TlvArray.js";
+import { TlvObject, TlvField } from "../../tlv/TlvObject.js";
+import { TlvEnum, TlvUInt32, TlvUInt16, TlvUInt64 } from "../../tlv/TlvNumber.js";
+import { TlvNullable } from "../../tlv/TlvNullable.js";
+import { TlvByteString } from "../../tlv/TlvString.js";
+import { BuildCluster } from "../../cluster/ClusterBuilder.js";
+
+/**
+ * @see {@link MatterCoreSpecificationV1_1} § 9.10.4.3
+ */
+export const enum AccessControlEntryAuthModeEnum {
+    Pase = 1,
+    Case = 2,
+    Group = 3
+};
+
+/**
+ * @see {@link MatterCoreSpecificationV1_1} § 9.10.4.4
+ */
+export const AccessControlTargetStruct = TlvObject({
```

Comment:
Sure will change, I haven't had a chance to look closely to see where I missed something in the coding conventions...  I was going to wait until I got the "OnOffCluster.with" stuff finished because depending on how I do the types there the generated code might change.  But if you point them out I can fix them.

Reply:
As said wehn reviewing in deptch I might find more ... but was just roughly reading over it now